src/disk_utils.cpp

```diff
@@ -1036,8 +1036,9 @@ void create_disk_layout(const std::string base_file, const std::string mem_index
 
 template <typename T, typename LabelT>
 int build_disk_index(const char *dataFilePath, const char *indexFilePath, const char *indexBuildParameters,
-                     diskann::Metric compareMetric, bool use_opq, bool use_filters, const std::string &label_file,
-                     const std::string &universal_label, const _u32 filter_threshold, const _u32 Lf)
+                     diskann::Metric compareMetric, bool use_opq, const std::string &codebook_prefix, bool use_filters,
```

Comment:
Should we bite the bullet and make all the const char*'s as std::string&?

Reply:
am OK, will DLVS or some other wrapper break.

Reply:
Oh, BANN might. The old DLVS still uses the "randnsg" sources, so it should be OK.